docs/ceph-mount-corruption.md

```diff
@@ -18,7 +18,8 @@ ls: /cephfs-share: Socket not connected
 or,
 
 ```
-# stat /home/kubelet/pods/ae344b80-3b07-4589-b1a1-ca75fa9debf2/volumes/kubernetes.io~csi/pvc-ec69de59-7823-4840-8eee-544f8261fef0/mount: transport endpoint is not connected
+# stat /home/kubelet/pods/ae344b80-3b07-4589-b1a1-ca75fa9debf2/volumes/kubernetes.io~csi/
```

Comment:
There are still quite a few blocks that contain console output. Maybe these need to be marked with

```
```console
```

for them to allow long lines?

Reply:
Updated.
Just putting this (```) will mark it as a code blocks, and it will be ignored.